hkm/templatetags/hkm_tags.py

```diff
@@ -125,3 +151,19 @@ def record_index(record, search_result):
 
     index = records_in_sr.index(record_in_sr) + 1
     return f"{index} / {search_result.get('resultCount')}"
+
+
+def trim_suffix(target, suffixes):
+    result = target
+    for suffix in suffixes:
+        if result.endswith(suffix):
+            result = result.split(suffix, 1)[0]
+    return result
+
+
+def trim_prefix(target, prefixes):
+    result = target
+    for prefix in prefixes:
+        if result.startswith(prefix):
+            result = result.split(prefix, 1)[1]
+    return result
```

Comment:
Starting from Python 3.9 there are [str.removeprefix()](https://docs.python.org/3.9/library/stdtypes.html#str.removeprefix) and [str.removesuffix()](https://docs.python.org/3.9/library/stdtypes.html#str.removesuffix).

Reply:
I was just thinking that can we rely on at least Python 3.9 being used. README.md doesn't explicitly say that at least Python 3.9 is required but it suggests to use at least Python 3.9 for non-Docker development. At least some of the GitHub actions use Python 3.9. I don't have the kuvaselaamo docker container running at the moment so I couldn't go into it and check what Python version is installed in the docker container which is actually the point here. But barring that I think it should be ok to rely on at least Python 3.9 being installed.

Reply:
https://github.com/City-of-Helsinki/kuvaselaamo/blob/master/Dockerfile#L2 🙂

Reply:
Ah! So it was there, missed that by looking at the docker-compose.yml, not the Dockerfile, and generally searching the codebase with "Python 3.". Python 3.9 it is in Docker also!